content/en/docs/faas/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Functions as a Service
+description: >-
+  OpenTelemetry supports various methods of monitoring Functions as a Service
+  provided by different cloud vendors
+weight: 2
+---
+
+Functions as a Service (FaaS) is an important serverless compute platform for
+cloud native applications. However, platform quirks usually mean these
+applications have slightly different monitoring guidance and requirements than
+applications running on Kubernetes or Virtual Machines.
+
+The initial vendor scope of the FaaS documentation is around Azure, GCP, and AWS
+functions (lambdas).
```

Comment:
```suggestion
functions (Lambdas).
```

you write Lambda with an uppercase L below, so I think this should be consistent